example-app/app/auth/reducers/auth.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { reducer } from './auth';
+import * as fromAuth from './auth';
+import { Login, LoginSuccess, Logout } from '../actions/auth';
+import { Authenticate, User } from '../models/user';
+
+describe('AuthReducer', () => {
+  describe('undefined action', () => {
+    it('should return the default state', () => {
+      const action = {} as any;
+
+      const result = reducer(undefined, action);
+      expect(result).toEqual(fromAuth.initialState);
+    });
+  });
+
+  describe('LOGIN_SUCCESS', () => {
+    it('should login a user', () => {
+      const user = { name: 'test' } as User;
```

Comment:
Preference, separate the arrange, act and asset sections of the test with new lines.